intake_stac/catalog.py

```diff
@@ -152,6 +153,22 @@ def _load(self):
     def _get_metadata(self, **kwargs):
         return kwargs
 
+    def to_geopandas(self):
+        """
+        Load the STAC Item Collection into a geopandas GeoDataFrame
+        """
+        try:
+            import geopandas as gpd
+        except ImportError:
+            raise ImportError(
+                "Using to_geopandas requires the `geopandas` package"
+            )
+
+        with tempfile.NamedTemporaryFile(suffix=".geojson") as f:
+            self._stac_obj.save(f.name)
+            gf = gpd.read_file(f.name)
```

Comment:
My main question about this PR is on the need for file serialization. Is this necessary or can we pass the geojson dict directly to Geopandas to create the dataframe?

Reply:
As @scotthq pointed out in https://github.com/pangeo-data/intake-stac/issues/36#issue-538136506, if you try to pass the geojson dict to Geopandas, you end up getting a `ValueError:`

```python
gpd.GeoDataFrame(results.items().geojson())
# Raises ValueError: arrays must all be same length
```
 I am open to suggestions on how to avoid the serialization step.

Reply:
@jhamman and @andersy005 

gf = gpd.GeoDataFrame.from_features(cat._stac_obj.geojson(), crs={'init' :'epsg:4326'})